examples/native_globals_redscript/Main.cpp

```diff
@@ -0,0 +1,111 @@
+#include <iostream>
+
+#include <RED4ext/RED4ext.hpp>
+#include <RED4ext/Scripting/Natives/Generated/Vector4.hpp>
+#include <RED4ext/Scripting/Natives/ScriptGameInstance.hpp>
+
+/*
+ * To run this plugin you need to load it with RED4ext (https://github.com/WopsS/RED4ext).
+ */
+
+void SummonVehiclePlease(RED4ext::IScriptable* aContext, RED4ext::CStackFrame* aFrame, void* aOut, int64_t a4)
+{
+    aFrame->code++; // skip ParamEnd
+
+    // Retrieve global game instance
+    RED4ext::ScriptGameInstance gameInstance;
+    RED4ext::Handle<RED4ext::IScriptable> handle;
+    // Get the player from the game instance
+    RED4ext::ExecuteGlobalFunction("GetPlayer;GameInstance", &handle, gameInstance);
+
+    if (handle)
+    {
+        auto rtti = RED4ext::CRTTISystem::Get();
+        // Get the PlayerPuppet class
+        auto playerPuppetCls = rtti->GetClass("PlayerPuppet");
+        // Set up the GetQuickSlotsManager function on the PlayerPuppet class
+        auto getQuickSlotsManagerFunc = playerPuppetCls->GetFunction("GetQuickSlotsManager");
+
+        RED4ext::Handle<RED4ext::IScriptable> quickSlotManager;
+        // Execute the function on our Player instance
+        RED4ext::ExecuteFunction(handle, getQuickSlotsManagerFunc, &quickSlotManager, {});
+
+        auto quickSlotsManagerCls = rtti->GetClass("QuickSlotsManager ");
+        auto summonVehicleFunc = quickSlotsManagerCls->GetFunction("SummonVehicle");
+
+        RED4ext::ExecuteFunction(quickSlotManager, summonVehicleFunc, nullptr, {});
+    }
+}
+
+void CustomFunctionTwo(RED4ext::IScriptable* aContext, RED4ext::CStackFrame* aFrame, void* aOut, int64_t a4)
+{
+    float var_one;
+    RED4ext::Vector4 var_two;
+
+    RED4ext::GetParameter(aFrame, &var_one);
+    RED4ext::GetParameter(aFrame, &var_two);
+
+    aFrame->code++; // skip ParamEnd
+}
+
+RED4EXT_C_EXPORT void RED4EXT_CALL RegisterTypes()
+{
+    // The function registration also works here - not sure which is better
```

Comment:
Can you please remove this line, registering functions should be done in `PostRegisterTypes`.